lib/lhm/migrator.rb

```diff
@@ -196,6 +196,7 @@ def destination_read
     end
 
     def index_ddl(cols, unique = nil, index_name = nil)
+      assert_valid_idx_name(index_name)
```

Comment:
super duper nitpick, why not  `validate_*` instead of `assert_*` I find that assert_ has a testing flavor in rubby